js/views/storeWizardVw.js

```diff
@@ -148,12 +150,19 @@ module.exports = baseModal.extend({
   },
 
   renderModerator: function(moderator){
+    //make sure this moderator is not a duplicate
```

Comment:
ok, i see what's happening. So, yeah this isn't data the client shares outside of this view. I was thinking userMd guids are being rendered (`this.model.get('user').moderator_guids`), but that's just the user's mods, whereas we're rendering all mods.

The code as is is alright.